packages/sdk/src/getTransferData/getTransferData.utils.ts

```diff
@@ -97,3 +112,37 @@ export async function getMin(config: AssetConfig, polkadot: PolkadotService) {
 
   return 0n;
 }
+
+interface ValidateSovereignAccountBalancesProps {
+  amount: bigint;
+  destination: DestinationChainTransferData;
+  source: SourceChainTransferData;
+}
+
+export function validateSovereignAccountBalances({
+  amount,
+  source,
+  destination,
+}: ValidateSovereignAccountBalancesProps): void {
+  const { sovereignAccountBalances } = destination;
```

Comment:
This will make conditions shorter and more readable.
```suggestion
  const { feeAssetBalance, transferAssetBalance  } = destination.sovereignAccountBalances;
```